### PR TITLE
{data}[GCCcore/10.3.0] Raptor v2.0.16

### PR DESCRIPTION
**Pull request by SIB-software**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by branfosj** [comment]
@boegelbot please test @ generoso

**Comment by branfosj** [comment]
Test report by @branfosj
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in total)
bear-pg0105u03a.bear.cluster - Linux RHEL 8.6, x86_64, Intel(R) Xeon(R) Platinum 8360Y CPU @ 2.40GHz (icelake), Python 3.6.8
See https://gist.github.com/branfosj/496d25378239b4d6dd6b8a6183caa6cb for a full test report.

File: easybuild/easyconfigs/r/Raptor/Raptor-2.0.16-GCCcore-10.3.0.eb [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+builddependencies = [
+    ('Autotools',  '20210128'),
+    ('make',       '4.3'),
+    ('binutils',   '2.36.1'),
+    ('libxml2',    '2.9.10'),
+    ('libxslt',    '1.1.34'),
+    ('cURL',       '7.76.0'),
+    ('ICU',        '69.1'),
+]
```
Comment (branfosj) [inline easybuild/easyconfigs/r/Raptor/Raptor-2.0.16-GCCcore-10.3.0.eb:34]: Some of these should be `dependencies` and not `builddependencies`.

**Review by branfosj — CHANGES_REQUESTED** [review]

File: easybuild/easyconfigs/r/Raptor/Raptor-2.0.16-GCCcore-10.3.0.eb (hunk 1 above)
Comment (smoretti) [reply easybuild/easyconfigs/r/Raptor/Raptor-2.0.16-GCCcore-10.3.0.eb]: Done

**Comment by smoors** [comment]
@boegelbot: please test @ generoso

**Comment by smoors** [comment]
@boegelbot please test @ jsc-zen2

**Review by smoors — APPROVED** [review]
lgtm

**Comment by smoors** [comment]
Going in, thanks @SIB-software!

**Pull request merged**